commit 10bb1b2fedc73e17387d51f33a296c122be7892d
Author: Theodore Ts'o <tytso@mit.edu>
Date:   Sat Feb 4 23:04:00 2017 -0500

    ext4: fix inline data error paths
    
    commit eb5efbcb762aee4b454b04f7115f73ccbcf8f0ef upstream.
    
    The write_end() function must always unlock the page and drop its ref
    count, even on an error.
    
    Signed-off-by: Theodore Ts'o <tytso@mit.edu>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1281_10bb1b2fedc73e17387d51f33a296c122be7892d.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1281_10bb1b2fedc73e17387d51f33a296c122be7892d.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
